packages/engine/src/scene/systems/ShadowSystem.tsx

```diff
@@ -482,3 +469,16 @@ export const ShadowSystem = defineSystem({
   execute,
   reactor
 })
+
+export const DropShadowSystem = defineSystem({
+  uuid: 'ee.engine.DropShadowSystem',
+  insert: { after: TransformSystem },
```

Comment:
this being after the TransformSystem means matrices are out of date. if we need another entity's matrix to be updated, we should use the ComputedTransformComponent to handle that relationship if we can